app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -26,6 +27,7 @@ describe('awsServerGroupCommandBuilder', function() {
 
     it('initializes to default values, setting usePreferredZone flag to true', function () {
       var command = null;
+      this.settings.providers.aws.defaults.iamRole = '{{application}}IAMRole';
```

Comment:
Sure ... I had just borrowed this pattern from the couple other places in `deck` that we're doing parameterized strings.

The particular format `{{application}}` is probably worth preserving as it's also used in `clouddriver` for other similar replacements.